GVFS/GVFS.Service/IRepoMounter.cs

```diff
@@ -0,0 +1,10 @@
+﻿using GVFS.Common.Tracing;
+
+namespace GVFS.Service
+{
+    public interface IRepoMounter
+    {
+        bool MountRepository(string repoRoot, int sessionId, ITracer tracer);

```

Comment:
Does this need to take in an `ITracer`?

Reply:
This method internally logs error rather than returning an `out string error`. I didn't change the pattern.

Reply:
Re-activating this conversation because I don't think the tracer should be here, but should be part of the object (and injected at construction time).

Reply:
Yes - that was what I meant originally. I didn't mean that the method shouldn't trace, but rather that the tracer should be available as an instance variable inside the method, and not passed in via the method.

Reply:
Removed `ITracer` from the interface. The implementation classes now take this in as a dependency in their Constructors.